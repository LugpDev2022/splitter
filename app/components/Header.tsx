const Header = () => (
  <header className='flex flex-col justify-center'>
    <h1
      className='
      text-3xl
      tracking-[.3em]
      font-semibold
      text-center
      text-[#00494d]
      '
    >
      SPLI
      <br />
      TTER
    </h1>
  </header>
);

export default Header;
