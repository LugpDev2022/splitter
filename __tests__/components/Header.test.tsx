import Header from '@/app/components/Header';
import { render } from '@testing-library/react';

describe('Tests on <Header />', () => {
  test('should match snapshot', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
