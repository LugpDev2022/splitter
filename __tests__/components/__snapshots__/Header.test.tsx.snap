// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <Header /> should match snapshot 1`] = `
<div>
  <header
    class="flex flex-col justify-center"
  >
    <h1
      class=" text-3xl tracking-[.3em] font-semibold text-center text-[#00494d] "
    >
      SPLI
      <br />
      TTER
    </h1>
  </header>
</div>
`;
